lib/presenters/artefact_presenter.rb

```diff
@@ -95,6 +95,10 @@ def present
       }
     end
 
+    if downtime = Downtime.for(@artefact)
+      presented['downtime_message'] = downtime.message if downtime.publicise?
+    end
```

Comment:
I'd have thought that we would want to include more than just the message in the returned JSON.  This would mean that the frontend can treat the messaging differently for upcoming and currently active downtime events for example.